decidim-proposals/lib/decidim/proposals/proposal_serializer.rb

```diff
@@ -131,7 +131,7 @@ def author_name(author)
       end
 
       def author_url(author)
-        if author.respond_to?(:nickname)
+        if author.respond_to?(:nickname) && !author.deleted?
```

Comment:
could you add a spec in decidim-proposals/spec/services/decidim/open_data_exporter_spec.rb ? 

Also , could you patch the profile_url method to look like: decidim-core/app/presenters/decidim/user_presenter.rb  ?